smart-contracts/test/test_bridgeBankLock.js

```diff
@@ -409,6 +485,19 @@ describe("Test Bridge Bank", function () {
     });
 
     it("should revert when multi-lock parameters are malformed, invalid sif addresses", async function () {
+      // Add the tokens into white list
+      await state.bridgeBank.connect(operator)
+        .updateEthWhiteList(state.token1.address, true)
+        .should.be.fulfilled;
+
+      await state.bridgeBank.connect(operator)
+        .updateEthWhiteList(state.token2.address, true)
+        .should.be.fulfilled;
+
+      await state.bridgeBank.connect(operator)
```

Comment:
Once we add the bulkWhitelist function, we can update these tests to only hit that one function.